sites/public/src/components/listings/ListingsCombinedDeprecated.module.scss

```diff
@@ -4,9 +4,10 @@
   @media (max-width: $screen-md) {
     display: block;
     position: absolute;
+    top: 0;
     height: 3.25rem;
     width: 100%;
-    z-index: 2;
+    z-index: var(--seeds-z-index-overlay);
```

Comment:
thought: I didn't know this variable existed and took a bit to track down it's value. Not for this PR but I wonder how we can make it easier for the whole team to use these sort of variables.

Reply:
Sounds good! Do you have the autocomplete variable extension installed? It will show the value in your IDE as well. https://github.com/bloom-housing/bloom/blob/main/sites/public/README.md